From 82b9d24e4dff7634fd8dacb3f41c5818557f9a99 Mon Sep 17 00:00:00 2001
From: Brendan Krieger <bkrieger@winsystems.com>
Date: Tue, 6 Apr 2021 16:38:01 +0000
Subject: [PATCH] Adding DTS to Makefile for C444

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index aa0c6a75c3..4493efc3e3 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -869,6 +869,7 @@ dtb-$(CONFIG_ARCH_IMX8M) += \
 	imx8mq-ddr3l-val.dtb \
 	imx8mq-ddr4-val.dtb \
 	imx8mq-evk.dtb \
+	imx8mq-itx-p-c444.dtb \
 	imx8mm-beacon-kit.dtb \
 	imx8mn-beacon-kit.dtb \
 	imx8mq-phanbell.dtb \
-- 
2.35.1

