From 38788ba07c97ffa2e2e5c89ea39a1e979a49c7f6 Mon Sep 17 00:00:00 2001
From: Brendan Krieger <bkrieger@winsystems.com>
Date: Tue, 6 Apr 2021 16:38:01 +0000
Subject: [PATCH] Adding DTS to Makefile for C444

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 45a79f02017..24595f8b149 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -993,6 +993,7 @@ dtb-$(CONFIG_ARCH_IMX8M) += \
 	imx8mq-ddr3l-val.dtb \
 	imx8mq-ddr4-val.dtb \
 	imx8mq-evk.dtb \
+	imx8mq-itx-p-c444.dtb \
 	imx8mm-beacon-kit.dtb \
 	imx8mn-beacon-kit.dtb \
 	imx8mq-mnt-reform2.dtb \
